jokes/jokes-router.js

```diff
@@ -1,8 +1,9 @@
 const axios = require('axios');
 
 const router = require('express').Router();
+const restricted = require('../auth/authenticate-middleware');
 
```

Comment:
you are already using the authenticate middleware to get to the jokes router on server.js. So you don't need it here necessarily.